test/PerformanceTests/ComponentTests/ODataReader/ODataReaderFeedTests.cs

```diff
@@ -44,49 +44,50 @@ public void Cleanup()
             _stream.Dispose();
         }
 
-        [IterationSetup(Target = nameof(ReadFeed))]
+        ////[IterationSetup(Target = nameof(ReadFeed))]

```

Comment:
What's the reason for/ramification of commenting these out?

Reply:
These tests are failing since they were converted to BenchmarkDotNet (confirmed from Clement). I have removed them for now since we aren't running these tests in any automation. This way, there is no confusion when someone observes these failing locally. They now will simply be skipped when the benchmarks are executed.

This is also the work that is being tracked by the [newly added work item](https://github.com/OData/odata.net/issues/2289) from the `.md` file